RDMMonitor.js

```diff
@@ -581,8 +706,35 @@ async function SendDMAlert(device)
     return;
 }
 
+function SendRebootAlert(device)
+{
+    if(!config.sendRebootAlerts)
+    {
+        return;
+    }
+    else
+    {

```

Comment:
```suggestion
    if(!config.sendRebootAlerts)
    {
        return;
    }
```
Remove unnecessary `else` statement